recipes/magnum/all/test_package/conanfile.py

```diff
@@ -25,10 +25,18 @@ def _executables(self):
                 pass
         return available
 
+    def _importer_plugins_folder(self):
+        magnum_plugin_libdir = "magnum-d" if self.settings.build_type == "Debug" else "magnum"
+        pkg_dir = "bin" if self.settings.os == "Windows" else "lib"
+        return os.path.join(self.deps_cpp_info["magnum"].rootpath, pkg_dir, magnum_plugin_libdir, "importers")
```

Comment:
maybe it's worth to add plug-ins folder to `user_info`?

Reply:
This is an amazing suggestion. Thanks! I'll use it also in the `magnum-plugins` PR